src/components/Community/data.json

```diff
@@ -1,8 +1,8 @@
 {
   "hero": {
-    "pictureDesktop": "/img/community/banner.png",
```

Comment:
@andronovhopf if we don't use these images anymore - we can delete them

Reply:
got it!

Reply:
done

Reply:
🙏